### ISSUE TITLE
Add card management examples

### ISSUE THREAD

**Issue by raoulmillais** [raoulmillais/node-7digital-api#45]

**Comment by cameronjroe** [comment]
Agreed. It's not very clear how to register a card with the `getCardRegistration` method. This seems to be the only function that needs to be passed an id param and I'm not sure what arguments `getCardRegistration` takes. Is there a definition somewhere in the docs? Or via the schema that's creating the API? Thanks!
